features/backend/deleting_shipping_methods.feature

```diff
@@ -0,0 +1,20 @@
+@shipping
+Feature: Deleting a shipping method
+  In order to remove not used or invalid shipping methods
+  As an Administrator
+  I want to be able to delete a shipping method
+
+  Background:
+    Given there is a shipping method "DHL Express"
```

Comment:
Actually we should follow: `store has a "X" shipping method ...` these steps are already implemented [here](https://github.com/Sylius/Sylius/blob/master/features/shop/apply_correct_shipping_fee_on_order.feature#L11) but probably needs tweaking.